tests/test_errors.py

```diff
@@ -254,7 +258,7 @@ def f(input_value, info):
         raise PydanticCustomError('my_error', 'my message', ctx)
 
     v = SchemaValidator(
-        {'type': 'function-before', 'function': {'type': 'general', 'function': f}, 'schema': {'type': 'int'}}
+        {'type': 'function-before', 'function': {'type': 'with-info', 'function': f}, 'schema': {'type': 'int'}}
```

Comment:
Same change all of these to use the functions

Reply:
mostly done.